pavement.py

```diff
@@ -6,7 +6,7 @@
 NAME = 'blackbelt'
 VERSION = '0.12.1'
 
-requires = ['click', 'requests', 'PyGithub', 'slacker', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
+requires = ['click', 'requests', 'PyGithub', 'slacker==0.6.2', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
```

Comment:
The idea is described here: https://caremad.io/2013/07/setup-vs-requirement/. Basically, `setup.py` is similar to `package.json`, `requirements.txt` is similar to shrinkwrap.

BB isn't deployable project, so in ideal world it should have only `setup.py` with as loosely pinned dependencies as possible.

And then there is Paver. I think it's something like Grunt, basically a task runner. Most of Python projects don't use it for managing dependencies. Specifying requirements in `pavement.py` is Paver-specific and I have no idea what to do with it.

What we should strive for is to have at least a single source of truth, single place where to put new dependencies.